### ISSUE TITLE
RPC end point for coin managements

### ISSUE THREAD

**Issue by gegaowp** [MystenLabs/sui#4323]
refer this doc 
https://www.notion.so/mystenlabs/Coin-management-02d1a156023049aeae31548212c75a6a

expose RPC end-points so that users can unblock themselves by manual merging if needed: merging or splitting coins to get a coin of at least or exact amount of Sui

This depends on corresponding implementations on the Move side

**Comment by gegaowp** [comment]
this is no longer relevant as we have separate pay paySui and PayAllSui endpoints tracking issue.
